app/views/browse/pod.html

```diff
@@ -142,10 +142,16 @@
 
                       <div class="mar-left-xl mar-bottom-lg">
                         <div class="row">
-                          <div class="pad-left-none pad-bottom-lg col-sm-6 col-lg-4">
+                          <div class="pad-left-none pad-bottom-md col-sm-6 col-lg-4">
```

Comment:
@jwforres, it will. It was an intentional change to tighten the page spacing up.

![image](https://cloud.githubusercontent.com/assets/19572090/18070743/ecc0b998-6e1d-11e6-99d1-973f96f58957.png)